src/js/script.js

```diff
@@ -1,5 +1,3 @@
-import 'babel-polyfill';
-
```

Comment:
Tried also with ie11 only, and then without it up until the moment uglify js yelled  at me because of es6 syntax. And I don't have infinite time to fight with it :D